clients/include/blas3/testing_herkx.hpp

```diff
@@ -87,140 +88,171 @@ void testing_herkx_bad_arg(const Arguments& arg)
-                                             lda,
-                                             dB,
-                                             ldb,
-                                             beta,
-                                             dC,
-                                             ldc),
-                              HIPBLAS_STATUS_INVALID_VALUE);
-        EXPECT_HIPBLAS_STATUS(hipblasHerkxFn(handle,
-                                             (hipblasFillMode_t)HIPBLAS_OP_N,
-                                             transA,
-                                             N,
-                                             K,
-                                             alpha,
-                                             dA,
-                                             lda,
-                                             dB,
-                                             ldb,
-                                             beta,
-                                             dC,
-                                             ldc),
-                              HIPBLAS_STATUS_INVALID_ENUM);
-        // EXPECT_HIPBLAS_STATUS(
-        //     hipblasHerkxFn(
-        //         handle, uplo, HIPBLAS_OP_T, N, K, alpha, dA, lda, dB, ldb, beta, dC, ldc),
-        //     HIPBLAS_STATUS_INVALID_VALUE);
-        EXPECT_HIPBLAS_STATUS(hipblasHerkxFn(handle,
-                                             uplo,
-                                             (hipblasOperation_t)HIPBLAS_FILL_MODE_FULL,
-                                             N,
-                                             K,
-                                             alpha,
-                                             dA,
-                                             lda,
-                                             dB,
-                                             ldb,
-                                             beta,
-                                             dC,
-                                             ldc),
-                              HIPBLAS_STATUS_INVALID_ENUM);
+        DAPI_EXPECT(HIPBLAS_STATUS_NOT_INITIALIZED,
+                    hipblasHerkxFn,
+                    (nullptr, uplo, transA, N, K, alpha, dA, lda, dB, ldb, beta, dC, ldc));
+
+        DAPI_EXPECT(
+            HIPBLAS_STATUS_INVALID_VALUE,
+            hipblasHerkxFn,
+            (handle, HIPBLAS_FILL_MODE_FULL, transA, N, K, alpha, dA, lda, dB, ldb, beta, dC, ldc));
+        DAPI_EXPECT(HIPBLAS_STATUS_INVALID_ENUM,
+                    hipblasHerkxFn,
+                    (handle,
+                     (hipblasFillMode_t)HIPBLAS_OP_N,
+                     transA,
+                     N,
+                     K,
+                     alpha,
+                     dA,
+                     lda,
+                     dB,
+                     ldb,
+                     beta,
+                     dC,
+                     ldc));
+        // DAPI_EXPECT(HIPBLAS_STATUS_INVALID_VALUE,
```

Comment:
Should the commented out code be removed?

Reply:
uncommented it and moved into `if(arg.bad_arg_all)` since cuBLAS documentation is unclear on whether or not this is supported.